doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1945,6 +2188,23 @@ Unstable NFC tag samples
 Cellular samples
 ================
 
+.. rst-class:: v2-5-0
+
+IRIS-7398: MCUBoot secondary partition in external flash fails for the nRF9161 DK
```

Comment:
```suggestion
IRIS-7398: The :ref:`nrf_cloud_multi_service` sample does not support using the MCUBoot secondary partition in external flash fails on the nRF9161 DK.
```

Reply:
Updated. Please check. We don't use a period at the end for the Known Issue headings.

Reply:
Looks good -- Using `and it fails on the nRF9161 DK`  instead of `on the nrf9161 DK` is potentially unclear, since it implies both statements are separately true, as opposed to the latter being a qualifier for the former, but it'll probably be alright